spec/unit/csv_builder_spec.rb

```diff
@@ -218,6 +223,13 @@ def view_context
       expect(builder).to receive(:build_row).and_return([]).once.ordered { |post| expect(post.id).to eq @post1.id }
       builder.build dummy_controller, []
     end
+
+    it "should generate data ignoring pagination" do
+      expect(dummy_controller).to receive(:find_collection).with({except: :pagination}).once.and_call_original
```

Comment:
Line is too long. [110/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.